seed/data_importer/tasks.py

```diff
@@ -253,11 +254,15 @@ def map_row_chunk(ids, file_pk, source_type, prog_key, **kwargs):
     save_type = PORTFOLIO_BS
     if source_type == ASSESSED_RAW:
         save_type = ASSESSED_BS
+    elif source_type == BUILDINGSYNC_RAW:
+        save_type = BUILDINGSYNC_RAW
 
     org = Organization.objects.get(pk=import_file.import_record.super_organization.pk)
 
     # get all the table_mappings that exist for the organization
     table_mappings = ColumnMapping.get_column_mappings_by_table_name(org)
+    if source_type == BUILDINGSYNC_RAW:
+        table_mappings['PropertyState']['_source_filename'] = ('PropertyState', '_source_filename', '', True)
```

Comment:
When later mapping additional models for bsync files in map_additional_models (BuildingFile, Scenario, Meters, ...) we need to re-read the file. If the imported file was a zip (ie multiple xml files), we thus need to know which file a PropertyState came from.

It's unfortunate to have to include this but I couldn't think of a better way. I decided to basically 'hardcode' this into all buildingsync table mappings here because it felt too strange for whoever was generating column mappings (presets or some buildingsync function) to be responsible for this weird import quirk. Looking for feedback and ideas